### PR TITLE
error for cookies

### PR DESCRIPTION
**Pull request by mhoehne**
onSubmit: (values) => {
      CreateAccount(values)
        .then((values) => {
          setCookie('email', emailAddress, { path: '/' });
          return navigate('/my-products');
        })
        .catch((msg) => {
          alert('error');
        });
    },

errors for: 

CreateAccount(values)
//Argument of type '{ _id: undefined; emailAddress: string; isAdmin: string; password: string; firstName: string; lastName: string; companyName: string; lastLogin: string; acceptedTermAndConditions: boolean; }' is not assignable to parameter of type 'AccountType'.

&

 setCookie('email', emailAddress, { path: '/' }); 
//Cannot find name 'emailAddress'

### CODE REVIEW

File: src/Pages/SignUpFormik.tsx [hunk 1]
```diff
@@ -29,102 +29,109 @@ const theme = createTheme();
...
+      lastName: '',
+      companyName: '',
+      lastLogin: '',
+      acceptedTermAndConditions: false,
+    },
+    validationSchema: validationSchema,
+    onSubmit: (values) => {
+      CreateAccount(values)
+        .then((values) => {
+          setCookie('email', emailAddress, { path: '/' });
```
Comment (nawarian) [inline src/Pages/SignUpFormik.tsx:127]: `emailAddress` does not exist in this scope. You probably want to get the email from the `values` variable instead.

Comment (mhoehne) [reply src/Pages/SignUpFormik.tsx:127]: so just `values` instead of `emailAddress`?

File: src/Pages/SignUpFormik.tsx [hunk 2]
```diff
@@ -29,102 +29,109 @@ const theme = createTheme();
...
+      password: '',
+      firstName: '',
+      lastName: '',
+      companyName: '',
+      lastLogin: '',
+      acceptedTermAndConditions: false,
+    },
+    validationSchema: validationSchema,
+    onSubmit: (values) => {
+      CreateAccount(values)
```
Comment (nawarian) [inline src/Pages/SignUpFormik.tsx:125]: Have a look at your [AccountType](https://github.com/mhoehne/ecozer.io/blob/main/src/API.ts#L14-L24) and then at your error:

> Argument of type '{ _id: undefined; emailAddress: string; **isAdmin: string**; password: string; firstName: string; lastName: string; companyName: string; lastLogin: string; acceptedTermAndConditions: boolean; }' is not assignable to parameter of type 'AccountType'.

Some of your field types don't match. Look at `isAdmin`, its return type and which type it has before attempting to create the account.

Comment (mhoehne) [reply src/Pages/SignUpFormik.tsx:125]: ah, got it. stupid me haha

File: src/Pages/SignUpFormik.tsx (hunk 1 above)
Comment (nawarian) [reply src/Pages/SignUpFormik.tsx:127]: No. `values` is probably of type `AccountType`, right? You can get the emailAddress field out of it.

**Pull request merged**